gears/src/baseapp/params.rs

```diff
@@ -35,8 +35,18 @@ const SEC_TO_NANO: i64 = 1_000_000_000;
 //##################################################################################
 //##################################################################################
 
+/// A domain ConsensusParams type that wraps domain consensus params types.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ConsensusParams {
+    pub block: Option<BlockParams>,
```

Comment:
Як я розумію всі поля структури обов'язкові

Reply:
I've inherited the tendermint structure. 

If we changed types to `T` instead of `Option<T>` we need to add default implementations. 

What do you think @joneskm ?

Reply:
The fields shouldn't be optional. We should have default implementations.

Reply:
fixed
Look at https://github.com/rumos-io/gears/pull/97/commits/fc68d2d8ae8cccbcbedf8d1a7cb52a904573cfcd, please